riocli/apply/manifests/device.yaml

```
@@ -42,4 +49,33 @@ spec:
     rosbagMountPath: "/opt/rapyuta/volumes/rosbag"
   preinstalled:
     enabled: True
-    catkinWorkspace: "/home/rapyuta/catkin_ws"
\ No newline at end of file
+    catkinWorkspace: "/home/rapyuta/catkin_ws"
+
+---
+apiVersion: "apiextensions.rapyuta.io/v1"
+kind: "Device"
+metadata:
+  name: "virtual-device-docker" # Required
+  project: "project-guid"
+  labels:
+    custom_label_1: label1
+    custom_label_2: label2
+    custom_label_3: label3
+    app: test
+spec:
+  python: "3"
+  rosDistro: "melodic" # Options: ["kinetic", "melodic" (default), "noetic"]
+  virtual:
+    enabled: True # Required
+    product: "sootballs" # Required Options: ["sootballs", "flaptter", "oks"]
+    arch: "amd64" # Options: ["amd64" (default), "arm64" ]
+    os: "ubuntu" # Options: ["ubuntu" (default), "debian" ]
+    codename: "focal" # Options: ["bionic", "focal" (default), "jammy", "bullseye"]
+    highperf: True # Optional [Ture, False (default)]
```

nit: If someone copy-pastes this manifest, highperf should be set to False.
```suggestion
    highperf: False # Optional [True, False (default)]
```